### DISCUSSION TITLE
On new pages I can not add the 4 pnp search webparts - they are not showing up

### DISCUSSION THREAD

**Discussion by hpreis** [microsoft-search/pnp-modern-search#2427] [Q&A]
Version: 4.8.0.0

Bug Description:
On new pages I can not add the 4 pnp search webparts to a page because they are not showing up, I can not select them.

What happened before:
1. I updated pnp-modern-search-parts-v4.sppkg from version 4.5.4.0 to last version 4.8.0.0 
2. I created a new site collection (publishing site)
3. I added the app "PnP Modern Search - Search Web Parts - v4" to the site collection
4. I set "Make this solution available to all sites in the organizazion" => false

When I turn on "Make this solution available to all sites in the organizazion" => true it works.

**Comment by hpreis** [comment]
I discovered something...
![grafik](https://user-images.githubusercontent.com/21243504/194304099-972877e8-54a4-4006-a47a-e728c3e226dc.png)

This shows only up when "Make this solution available to all sites in the organizazion" => true 

German: "Wenn Sie diese Einstellung deaktivieren, können Benutzer das Webpart nicht zu Seiten hinzufügen. Falls das Webpart bereits zu Seiten hinzugefügt war, wird es weiterhin funktionieren."

English: "**If you clear this setting, users won't be able to add the web part to pages. The web part will continue to work if it was already added to pages.**"

But [here](https://microsoft-search.github.io/pnp-modern-search/installation/) it's stated that it's not mandatory

![grafik](https://user-images.githubusercontent.com/21243504/194305067-b5af9afa-113f-4465-bf53-897b8e621dbb.png)

- For the pnp-modern-search-parts.sppkg package, you can choose to make the solution available in [all sites](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/tenant-scoped-deployment) or force to install an app to the site every time.

**Reply by wobba** [reply]
Your issue SPFx related, not web part related, so moved it to a discussion. It's not mandatory to check. If you don't check it you need to add the solution to the site you want to use the web parts before they are available on a page in that site.

**Reply by hpreis** [reply]
But that's the Point:
- I don't check "Make this solution available to all sites in the organization"
- I add the solution to a newly created publishing site
- then I want to add the pnp webparts to the main page => but they are not listed, I don't find them

Has anyone experienced the same behaviour?

**Reply by wobba** [reply]
Could be many weird SPFx reasons.

I would try to delete the app from the app catalog, and re-add it as step 1. And remove and re-add from the site as step 2.
